pyproject.toml

```diff
@@ -31,6 +31,8 @@ dependencies = [
     'fonticon-materialdesignicons6',
     'qtpy',
     'tifffile',
+    'PyYAML',
```

Comment:
adding pyyaml just to store data in a yaml file that could easily be stored as a dict is a no-no.